### ISSUE TITLE
arcore shared library is not stripped

### ISSUE THREAD

**Issue by silentnuke** [google-ar/arcore-android-sdk#304]
It looks like shared library which is shipped with arcoresdk is not stripped. Inside libarcore_sdk.so there are two sections that shouldn't be there if the library is stripped (.symtab and .strtab).

Here is an example for `armeabi-v7a`
```
-rw-r--r--@ 1 kostiantyn.zghirovskyi  staff  613060 Feb 21 09:05 libarcore_sdk.so
```
```
kostiantyn.zghirovskyi$ `ndk-which readelf` -S libarcore_sdk.so 
There are 28 section headers, starting at offset 0x95664:

Section Headers:
  [Nr] Name              Type            Addr     Off    Size   ES Flg Lk Inf Al
  [ 0]                   NULL            00000000 000000 000000 00      0   0  0
  [ 1] .dynsym           DYNSYM          000000f4 0000f4 001a20 10   A  2   1  4
  [ 2] .dynstr           STRTAB          00001b14 001b14 00259c 00   A  0   0  1
  [ 3] .hash             HASH            000040b0 0040b0 000aac 04   A  1   0  4
  [ 4] .gnu.version      VERSYM          00004b5c 004b5c 000344 02   A  1   0  2
  [ 5] .gnu.version_d    VERDEF          00004ea0 004ea0 000038 00   A  2   2  4
  [ 6] .gnu.version_r    VERNEED         00004ed8 004ed8 000070 00   A  2   3  4
  [ 7] .rel.dyn          REL             00004f48 004f48 003bb8 08   A  1   0  4
  [ 8] .rel.plt          REL             00008b00 008b00 0006f8 08   A  1   0  4
  [ 9] .plt              PROGBITS        000091f8 0091f8 000a88 00  AX  0   0  4
  [10] .text             PROGBITS        00009c80 009c80 0316f0 00  AX  0   0 16
  [11] malloc_hook       PROGBITS        0003b370 03b370 0003dc 00  AX  0   0  4
  [12] .ARM.extab        PROGBITS        0003b74c 03b74c 0044bc 00   A  0   0  4
  [13] .ARM.exidx        ARM_EXIDX       0003fc08 03fc08 0025e8 08  AL 10   0  4
  [14] .rodata           PROGBITS        000421f0 0421f0 009faa 00   A  0   0 16
  [15] .dynamic          DYNAMIC         0004d19c 04c19c 000118 08  WA  2   0  4
  [16] .data             PROGBITS        0004d2c0 04c2c0 000130 00  WA  0   0 16
  [17] .fini_array       FINI_ARRAY      0004d3f0 04c3f0 000008 00  WA  0   0  4
  [18] .init_array       INIT_ARRAY      0004d3f8 04c3f8 000050 00  WA  0   0  4
  [19] .data.rel.ro      PROGBITS        0004d450 04c450 0020ac 00  WA  0   0 16
  [20] .got              PROGBITS        0004f4fc 04e4fc 0006ac 00  WA  0   0  4
  [21] .bss              NOBITS          0004fbc0 04ebc0 0027ac 00  WA  0   0 64
  [22] .comment          PROGBITS        00000000 04eba8 000065 01  MS  0   0  1
  [23] .note.gnu.gold-ve NOTE            00000000 04ec10 00001c 00      0   0  4
  [24] .ARM.attributes   ARM_ATTRIBUTES  00000000 04ec2c 00003a 00      0   0  1
  [25] .symtab           SYMTAB          00000000 04ec68 01fd50 10     26 7720  4
  [26] .strtab           STRTAB          00000000 06e9b8 026b9f 00      0   0  1
  [27] .shstrtab         STRTAB          00000000 095557 00010c 00      0   0  1
```

```
kostiantyn.zghirovskyi$ `ndk-which strip` -g -S -d --strip-debug --strip-unneeded libarcore_sdk.so
```
```
-rw-r--r--  1 kostiantyn.zghirovskyi  staff  323956 Apr 10 19:54 libarcore_sdk.so
```
```
kostiantyn.zghirovskyi$ `ndk-which readelf` -S libarcore_sdk.so 
There are 26 section headers, starting at offset 0x4ed64:

Section Headers:
  [Nr] Name              Type            Addr     Off    Size   ES Flg Lk Inf Al
  [ 0]                   NULL            00000000 000000 000000 00      0   0  0
  [ 1] .dynsym           DYNSYM          000000f4 0000f4 001a20 10   A  2   1  4
  [ 2] .dynstr           STRTAB          00001b14 001b14 00259c 00   A  0   0  1
  [ 3] .hash             HASH            000040b0 0040b0 000aac 04   A  1   0  4
  [ 4] .gnu.version      VERSYM          00004b5c 004b5c 000344 02   A  1   0  2
  [ 5] .gnu.version_d    VERDEF          00004ea0 004ea0 000038 00   A  2   2  4
  [ 6] .gnu.version_r    VERNEED         00004ed8 004ed8 000070 00   A  2   3  4
  [ 7] .rel.dyn          REL             00004f48 004f48 003bb8 08   A  1   0  4
  [ 8] .rel.plt          REL             00008b00 008b00 0006f8 08  AI  1  20  4
  [ 9] .plt              PROGBITS        000091f8 0091f8 000a88 00  AX  0   0  4
  [10] .text             PROGBITS        00009c80 009c80 0316f0 00  AX  0   0 16
  [11] malloc_hook       PROGBITS        0003b370 03b370 0003dc 00  AX  0   0  4
  [12] .ARM.extab        PROGBITS        0003b74c 03b74c 0044bc 00   A  0   0  4
  [13] .ARM.exidx        ARM_EXIDX       0003fc08 03fc08 0025e8 08  AL 10   0  4
  [14] .rodata           PROGBITS        000421f0 0421f0 009faa 00   A  0   0 16
  [15] .dynamic          DYNAMIC         0004d19c 04c19c 000118 08  WA  2   0  4
  [16] .data             PROGBITS        0004d2c0 04c2c0 000130 00  WA  0   0 16
  [17] .fini_array       FINI_ARRAY      0004d3f0 04c3f0 000008 04  WA  0   0  4
  [18] .init_array       INIT_ARRAY      0004d3f8 04c3f8 000050 04  WA  0   0  4
  [19] .data.rel.ro      PROGBITS        0004d450 04c450 0020ac 00  WA  0   0 16
  [20] .got              PROGBITS        0004f4fc 04e4fc 0006ac 00  WA  0   0  4
  [21] .bss              NOBITS          0004fbc0 04eba8 0027ac 00  WA  0   0 64
  [22] .comment          PROGBITS        00000000 04eba8 000065 01  MS  0   0  1
  [23] .note.gnu.gold-ve NOTE            00000000 04ec10 00001c 00      0   0  4
  [24] .ARM.attributes   ARM_ATTRIBUTES  00000000 04ec2c 00003a 00      0   0  1
  [25] .shstrtab         STRTAB          00000000 04ec66 0000fc 00      0   0  1
```

**Comment by inio** [comment]
Thanks for pointing this out.  We actually already noticed this ourselves and are working on improving this.

**Comment by inio** [comment]
This should be fixed with 1.2 which we announced today today.
